test/e2e_node/environment/setup_host.sh

```diff
@@ -62,6 +62,25 @@ if [ $? -ne 0 ]; then
   rm -r etcd-v2.2.5-linux-amd64*
 fi
 
+# Install nsenter for ubuntu images
+cat /etc/*-release | grep "ID=ubuntu"
+if [ $? -eq 0 ]; then
+  if ! which nsenter > /dev/null; then
+     echo "Do not find nsenter. Install it."
+     mkdir -p /tmp/nsenter-install
+     cd /tmp/nsenter-install
+     curl https://www.kernel.org/pub/linux/utils/util-linux/v2.24/util-linux-2.24.tar.gz | tar -zxf-
```

Comment:
If we want to trust jpetazzo / Dockerhub (arbitrary code execution as a service :) there's https://github.com/jpetazzo/nsenter#how-do-i-install-nsenter-with-this